pyannote/audio/applications/change_detection.py

```diff
@@ -0,0 +1,420 @@
+# all copies or substantial portions of the Software.
+
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+# AUTHORS
+# Ruiqing YIN
+
+"""
+change detection
+
+Usage:
+    pyannote-change-detection train [--database=<db.yml> --subset=<subset>] <experiment_dir> <database.task.protocol>
+    pyannote-change-detection evaluate [--database=<db.yml> --subset=<subset> --epoch=<epoch> --min_duration=<min_duration>] <train_dir> <database.task.protocol>
+    pyannote-change-detection apply  [--database=<db.yml> --subset=<subset> --threshold=<threshold> --epoch=<epoch>  --min_duration=<min_duration>] <train_dir> <database.task.protocol>
+    pyannote-change-detection -h | --help
+    pyannote-change-detection --version
+
+Options:
+    <experiment_dir>           Set experiment root directory. This script expects
+                               a configuration file called "config.yml" to live
+                               in this directory. See "Configuration file"
+                               section below for more details.
+    <database.task.protocol>   Set evaluation protocol (e.g. "Etape.SpeakerDiarization.TV")
+    <wav_template>             Set path to actual wave files. This path is
+                               expected to contain a {uri} placeholder that will
+                               be replaced automatically by the actual unique
+                               resource identifier (e.g. '/Etape/{uri}.wav').
+    <train_dir>                Set path to the directory containing pre-trained
+                               models (i.e. the output of "train" mode).
+    <tune_dir>                 Set path to the directory containing optimal
+                               hyper-parameters (i.e. the output of "tune" mode).
+    --database=<db.yml>        Path to database configuration file.
+                               [default: ~/.pyannote/db.yml]
+    --subset=<subset>          Set subset (train|developement|test).
+                               In "train" mode, default is "train".
+                               In "validation" mode, default is "development".
+                               In "tune" mode, default is "development".
+                               In "apply" mode, default is "test".
+    --epoch=<epoch>            The epoch in training process
+    --threshold=<threshold>    Threshold for choosing change points 
+                               [default: 0.1]
+    --min_duration=<min_duration>]  min duration between two adjacent peaks
+                                    [default: 1.0]
```

Comment:
You are right, I should align everything.  By the way, I've tested with `%run` in `jupyter notebook` and it works.